examples/contrib/DALI/dali_example_utilities.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Should we keep this copyright here ?

Reply:
I do need to in that particular file, as NVIDIA is my employer. They let me contribute to OSS on my own time though, provided I put that on new _plaintext_ files where I did more stuff than trivial bug fixes

Ironically, that's the one file where there is 0% NVIDIA stuff too

(I'm not on the DALI team, nor am I in engineering though, just a random Ignite fan - so I unfortunately am not gonna have a whole lot of deep DALI insights)

Reply:
Ok for the copyright line :) So, more precisely `make_keras_like` method brings the copyright ?

As you are at Nvidia, could you see with DALI team whether could be interested in the enhancements from `dali_transform_utilities.py` ?

Reply:
I will reach out!

Reply:
I spoke to DALI team and opened an issue there, so will mark the one here as resolved for now: https://github.com/NVIDIA/DALI/issues/1731